cmd/ctr/snapshot.go

```diff
@@ -195,22 +196,31 @@ var removeSnapshotCommand = cli.Command{
 
 var prepareSnapshotCommand = cli.Command{
 	Name:      "prepare",
-	Usage:     "prepare gets mount commands for digest",
-	ArgsUsage: "[flags] <digest> <target>",
-	Flags:     []cli.Flag{},
+	Usage:     "prepare a snapshot from a tar file by digest",
+	ArgsUsage: "[flags] digest",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "target",
```

Comment:
Sorry my previous comment may have been confusing. I think the usage should still be `[flags] digest target`, I just want an optional flag for `snapshot-name` that is used to set the ID of the snapshot that is created. So it would be possible to do something like...
```
$ ctr snapshot prepare --snapshot-name my-local-container sha256:dc1fff8aaab605d5223d9ee316cd1459e809306ce43ab81aa95f83fbc7763759 /path/to/desired-root
```
... to get the mounts, and also ...
```
$ ctr snapshot mounts --snapshot-name existing-snapshot-name /localpath/i/want/to/mount/to
```

My thought being that these commands can be used to mount snapshots and manipulate them rather than just administrating snapshots and mirroring the snapshotter interface.